### ISSUE TITLE
Direct usage of Module Augmentation with @lightningjs/core

### ISSUE THREAD

**Issue by itsjamie** [rdkcentral/Lightning#480]
It appears there is no way to augment the FireAncestors map with @lightningjs/core.

Because it exports `default` module augmentation is broken.

See: https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation
> Default exports also cannot be augmented, only named exports (since you need to augment an export by its exported name, and default is a reserved word - see [#14080](https://github.com/Microsoft/TypeScript/issues/14080) for details)

**Comment by frank-weindel** [comment]
Thanks for bringing this up @itsjamie. Just for anyone else who might stumble upon this, this problem only exists if you use `@lightningjs/core` without also utilizing `@lightningjs/sdk` which exports `Lightning` as a named export.

I've got a workaround for this that should've been documented (better yet if we can export `Lightning` as a named export). You can create a file that re-exports `Lightning` as a named export and base your augmentation file on it:

**lightning-reexport.ts**
```ts
import Lightning from '@lightningjs/core';
export { Lightning };
```

**augmentations.d.ts**
```ts
declare module './lightning-reexport' {
  /**
   * Lightning Core Augmentations
   */
  namespace Lightning {
    namespace Component {
      // ...
    }
    // ...
  }
  //...
}
```

The nice thing with this is that you won't have to change the way you import Lightning Core in the rest of your application. TypeScript applies the augmentations to the actual source types irregardless as to how they are imported.

**Comment by frank-weindel** [comment]
We are going to add a named export along side the default export

**Comment by frank-weindel** [comment]
The original PR for this issue was reverted. It is now resolved as part of #498.

**Comment by uguraslan** [comment]
Hey @frank-weindel, thanks for your comment and I am sorry about my confusion about this issue and #483. I have removed this from the July release as the fix has not been approved & merged yet. I hope we can include this one in the October release.
